dataland-frontend/src/components/forms/parts/fields/YesNoFormField.vue

```diff
@@ -159,6 +161,22 @@ export default defineComponent({
         return this.qualityOptions.filter((qualityOption) => qualityOption.value !== QualityOptions.Na);
       }
     },
+    reportsName(): string[] {
+      if (this.injectReportsNameAndReferences) {
```

Comment:
These two code blocks appear 4 times in total, they could be functions with input parameters

Reply:
Working on this

Reply:
Refactored methods are pushed